### DISCUSSION TITLE
mvn clean verify doesn't work after 1st clone

### DISCUSSION THREAD

**Discussion by Arnaud80** [Decathlon/tzatziki#10] [Q&A] [answered]
Hello,

I have this error just after cloning the project :

I tried to put args --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED directly in the JVM, but I get the same error.

I'm building with IntellJ on Windows 11.

```
[ERROR] Failures: 
[ERROR]   EnvTest.export:11 You are attempting to set an Environment variable at runtime but your security settings didn't allow it!
You need to add the following flags to your java command line or your surefire plugin argline for this to work:
--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED
```

Thanks in advance for your help.

**Comment by brian-mulier** [comment]
Hello !

I am able to reproduce the issue.
As of now, I can't spend much time finding a workaround, it looks like the JVM is not taking the args into account.
While I find a workaround you can build the lib by skipping the tests and run the tests aside as a JUnit / Cucumber run configuration with Intellij for eg. if you want to contribute and test it (that's what I am currently doing).

It could also be linked to Windows since @mimfgg is using MacOS and have no issue.

Sorry for the inconvenience !

**Reply by Arnaud80** [reply]
Thank you for your reply Brian and your tests.
Without any action from my side, it's working now. It's the magic of Windows ;-)

**Reply by brian-mulier** [reply]
Okay great, I will investigate anyway because I don't really like magic 🧙

**Comment by brian-mulier** [comment, marked as answer]
It is now fixed and no longer needs any flag in version 1.0.4 ! :)
